package.json

```diff
@@ -98,7 +98,7 @@
     "@emotion/styled": "11.11.5",
     "@loadable/component": "5.16.4",
     "@loadable/server": "5.16.5",
-    "@optimizely/react-sdk": "2.9.2",
+    "@optimizely/react-sdk": "3.2.2",
```

Comment:
Worth keeping an eye on analytics for Opera Mini once we merge this in. I know its been tested on Opera Mini Android with Extreme Data saving on, but it would be good to ensure the analytics don't drop off in production as the version of Opera Mini used by the audience is different.

Reply:
Ah yh good point - I only tested the version on my Google Pixel 6 so can imagine that is a bit biased